hoomd/md/constrain.py

```diff
@@ -44,73 +44,55 @@ def _attach(self):
         super()._attach()
 
 
-class distance(Constraint):
-    R"""Constrain pairwise particle distances.
+class Distance(Constraint):
+    """Constrain pairwise particle distances.
 
-    :py:class:`distance` specifies that forces will be applied to all particles
-    pairs for which constraints have been defined.
+    Args:
+        tolerance (float): Relative tolerance for constraint violation warnings.
 
-    The constraint algorithm implemented is described in:
+    `Distance` applies constraint forces between particles to satisfy
+    user-specified distance constraints. The algorithm implemented is described
```

Comment:
Do the distance constraints fix a distance between particles, or do they prevent the distance between particles from exceeding a maximum value? This could be clarified somewhere in this docstring to make this class's purpose clearer.

Reply:
This implements a set of equality constraints using Lagrange multipliers. I have reworded the documentation to clarify.

Constraints based on inequalities are much more complex to solve. Many MD engines provide fixed distance constraints as some force fields specify this instead of using harmonic bonds.